routing/cross_mwm_road_graph.cpp

```diff
@@ -5,9 +5,41 @@
 
 namespace
 {
+using namespace routing;
+
 inline bool IsValidEdgeWeight(EdgeWeight const & w) { return w != INVALID_EDGE_WEIGHT; }
+
+void FindIngoingCrossNode(CrossRoutingContextReader const & currentContext, CrossNode const & toNode,
+                          IngoingCrossNode & ingoingNode)
+{
+  bool found = false;
+  auto const findingFn = [&ingoingNode, &toNode, &found](IngoingCrossNode const & node){
+    if (node.m_nodeId == toNode.node)
```

Comment:
Could you please explain, what happens here when the number of ingoing nodes near point is more than one, and why we select the only one?